_pages/en_US/formatting-sd-(linux).md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Formatting SD (Linux)"
+---
+
+{% include toc title="Table of Contents" %}
+
+### Required Reading
+
+This is an add-on section for formatting an SD card to work with the 3DS.
+
+If the 3DS already recognizes the SD card, this guide is not required.
+
+This page is for Linux users only. If you are not on Linux, check out the [Formatting SD (Windows)](formatting-sd-(Windows)) or [Formatting SD (Mac)](formatting-sd-(mac)) pages.
+
+### What You Need
+
+* The latest version of [guiformat](http://www.ridgecrop.demon.co.uk/index.htm?guiformat.htm)
+
+### Instructions
+#### Section I - Determining which slot your SD card is in
+
+1. Make sure your SD card is **not** inserted.
+2. Launch the Linux Terminal
+3. Type `watch "dmesg | tail -10"`
+4. Insert your SD card into your PC.
+5. Observe the output. It should match something like this:
+```
+[11599.200739] sd 3:0:0:0: [sdc] 7744512 512-byte logical blocks: (3.96 GB/3.69
+GiB)
+[11599.204630] sd 3:0:0:0: [sdc] Assuming drive cache: write through
+[11599.210491] sd 3:0:0:0: [sdc] Assuming drive cache: write through
+[11599.210495]  sdc:
+```
```

Comment:
Please note, for computers that have sdcard readers, like some laptops, the output may come like this:
```
[25687.697250] mmc0: new high speed SDHC card at address b368
[25687.733139] mmcblk0: mmc0:b368 00001 3.73 GiB (ro)
[25687.744772]  mmcblk0: p1
```
Maybe using `lsblk` would be better? Run before and after inserting card.
Also ensuring that `RO` is not 1 on the sdcard.

Reply:
To be honest, I don't have a stable way of accessing an SD card on my computers, so I can't really write up this part. One is unusable and the other is unusable for transfers since it disconnects every two seconds. Would you like to handle it for me?

I think if you fork my fork, edit it and send a PR, your name would come up here too. I'm not sure though if PRs work like that.

Reply:
Never tried to PR a fork trying to PR to the repo, but I'll give it a try, just a moment.